static/app/views/insights/pages/platform/shared/trafficWidget.tsx

```diff
@@ -139,7 +139,7 @@ export function TrafficWidget({
             }}
             onOpenFullScreen={() => {
               openInsightChartModal({
-                title: t('Requests'),
+                title: tct('[title]', {title}),
```

Comment:
I've never seen a translation this dynamic - usually it's defined somewhere, exported, and reused...but why not?

Reply:
This does not make sense. If it is fully dynamic it cannot be translated.
The title that is passed as prop should already use the `t` util to create a translated string.
```suggestion
                title,
```

Reply:
Oh true, didn't put 2 and 2 together there, thanks for spotting